engine/src/main/java/org/terasology/rendering/dag/RenderTaskListGenerator.java

```diff
@@ -54,20 +55,53 @@ private void logList(String title, List<?> list) {
     private void generateTaskList() {
```

Comment:
I'm finding this method a bit confusing. It's now a bit too late (2:54am) but I will give you specific feedback Saturday night at the latest.